components/popover/src/Popover.js

```diff
@@ -1,19 +1,25 @@
 /** @jsx jsx */
 
-import React, { useState, useEffect, useRef } from 'react';
+import { useState, useEffect, useRef, Fragment } from 'react';
 import PropTypes from 'prop-types';
-import { jsx } from '@westpac/core';
+import { jsx, useBrand, merge } from '@westpac/core';
+import pkg from '../package.json';
```

Comment:
I think this is making the build cranky

Reply:
I just need to run prettier on the package.json. Always happens when I do a yarn add, prettier doesnt like the way yarn formats adding a package. Ill make sure it passes build once I'm finished updating this component with overwrite tokens :smile: